ninja.lua

```diff
@@ -462,11 +473,15 @@ local function compilation_rules(cfg, toolset, pch)
 		p.outln("  deps = gcc")
 		p.outln("")
 		p.outln("RESFLAGS = " .. all_resflags)
-		p.outln("rule rc")
-		p.outln("  command = " .. rc .. " -i $in -o $out $RESFLAGS")
-		p.outln("  description = rc $out")
-		p.outln("")
-		if cfg.kind == p.STATICLIB then
+
+		if rc then
```

Comment:
We should also ensure that the rule is not called when unsupported.
and warn/error when it is the case.